src/m_argv.c

```diff
@@ -131,6 +131,11 @@ static void LoadResponseFile(int argv_index, const char *filename)
 
     for (i=0; i<argv_index; ++i)
     {
+        if (i == MAXARGVS)
+        {
+            I_Error("Too many arguments!");
```

Comment:
All three error cases have identical message. If someone actually reports hitting this we will have a hard time narrowing down which one was hit.

Also the message should report how many arguments we hit and what the max is.